src/adhocracy_frontend/adhocracy_frontend/static/js/Packages/Mapping/Mapping.ts

```diff
@@ -413,6 +428,17 @@ export var mapListingInternal = (adhConfig : AdhConfig.IService, adhHttp : AdhHt
                 scope.toggleItem(scope.items[index]);
                 scrollToItem(index);
             };
+
+            scope.resetMap = () => {
+                map.fitBounds(scope.polygon.getBounds());
+
+                // this is hacky but I could not find how to add
+                // a callback function to fitbounds as this always
+                // triggers the moveend event.
+                $timeout(() => {
+                    scope.showZoomButton = false;
+                }, 300);
```

Comment:
Maybe we could use [`LatLngBound.contains(<LatLngBounds>)`](http://leafletjs.com/reference.html#latlngbounds) in the moveend handler in order to only set this to `true` if necessary.